core/src/main/java/nl/nn/adapterframework/configuration/ClassNameRewriter.java

```diff
@@ -0,0 +1,46 @@
+/*
+   Copyright 2023 WeAreFrank!
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package nl.nn.adapterframework.configuration;
+
+import org.xml.sax.Attributes;
+import org.xml.sax.ContentHandler;
+import org.xml.sax.SAXException;
+
+import nl.nn.adapterframework.xml.FullXmlFilter;
+import nl.nn.adapterframework.xml.WritableAttributes;
+
+public class ClassNameRewriter extends FullXmlFilter {
+
+	public static final String LEGACY_PACKAGE_NAME = "nl.nn.adapterframework.";
+	public static final String ORG_FRANKFRAMEWORK_PACKAGE_NAME = "org.frankframework.";
+	public static final String CLASS_NAME_ATTRIBUTE = "className";
+
+	public ClassNameRewriter(ContentHandler handler) {
+		super(handler);
+	}
+
+	@Override
+	public void startElement(String uri, String localName, String qName, Attributes attributes) throws SAXException {
+		WritableAttributes writableAttributes = new WritableAttributes(attributes);
+		String className = writableAttributes.getValue(CLASS_NAME_ATTRIBUTE);
```

Comment:
@nielsm5 
Ik kijk hier naar de qualified name v/h attribuut, zonder gebruik van namespaces.
Dat gaat tot nu toe goed met onze test-files maar ik vraag me wel af of ik hier ook rekening moet houden met namespaces, en juist zoeken op localName v/h attribuut. (Maar dan zouden er meerdere attributen `className` kunnen zijn in meerdere namespaces, in theorie).

Reply:
Niemand gebruikt namespaces.. 😄 
Laat ik het anders zeggen, ik heb het nog nooit gezien. Mocht men dat wel doen, vraag ik mij af waarom, en moeten ze dat zelf aanpassen.